etc/notes/CHANGES_5.0.0.md

```diff
@@ -18,6 +18,84 @@ The following is a detailed collection of the changes in the major v5 release of
 
 ### Optional callback support migrated to `mongodb-legacy`
 
+Node v5 drops support for callbacks in favor of a pure async-await API.  Below are some strategies for
```

Comment:
```suggestion
Node v5 drops support for callbacks in favor of an async-await API.  Below are some strategies for
```

I don't think this has general meaning externally, its terminology we invented to differentiate using the syntax and making promises ourselves. It might also be more accurate to change "favor of an async-await API" to "favor of a Promise only API"